### PR TITLE
CommonServerPython - Added timeline object to the CommandResult object

### PR DESCRIPTION
**Pull request by bakatzir**
## Status
- [x] In Progress

## Related Issues
fixes: https://github.com/demisto/etc/issues/24788

## Does it break backward compatibility?
   - [x] No

## Must have
- [ ] Tests - will follow. 
- [x] Documentation

### CODE REVIEW

**Comment by bakatzir** [comment]
@anara123 Tests will follow, please let me know if this is what you had in mind.

File: Packs/Base/Scripts/CommonServerPython/CommonServerPython.py [hunk 1]
```diff
@@ -2600,12 +2634,15 @@ class CommandResults:
...
+    param timeline: must be a list. used by the server to populate an indicator's timeline
+
     :return: None
     :rtype: ``None``
     """
     def __init__(self, outputs_prefix=None, outputs_key_field=None, outputs=None, indicators=None, readable_output=None,
-                 raw_response=None):
-        # type: (str, object, object, list, str, object) -> None
+                 raw_response=None, indicators_timeline=None):
+        # type: (str, object, object, list, str, object, list) -> None
```
Comment (anara123) [inline Packs/Base/Scripts/CommonServerPython/CommonServerPython.py:2645]: should not `indicators_timeline` should be of type `IndicatorsTimeline`

**Review by yuvalbenshalom — COMMENTED** [review]
code looks good.
plase link to the docs of this issue

File: Packs/Base/Scripts/CommonServerPython/CommonServerPython.py [hunk 2]
```diff
@@ -2573,6 +2573,40 @@ def to_context(self):
...
+            indicators = []
+        else:
+            indicators = [indicators] if isinstance(indicators, dict) else indicators
+
+        for indicator in indicators:
+            if category:
+                indicator['Category'] = category
+            elif 'Category' not in indicator.keys():
+                indicator['Category'] = 'Integration Update'
+            if message:
```
Comment (yaakovi) [inline Packs/Base/Scripts/CommonServerPython/CommonServerPython.py:2604]: add an empty line before this condition

**Review by yaakovi — APPROVED** [review]

**Comment by bakatzir** [comment]
Opening a new PR https://github.com/demisto/content/pull/8868

**Pull request closed without merging**